### DISCUSSION TITLE
请问哪里可以看到我的服务器上所有用户的访问记录

### DISCUSSION THREAD

**Discussion by bigbigbigstone** [XTLS/Xray-core#931] [Q&A]
请问如何设置才能看到自己X-Ray服务器上的用户访问记录或日志，包括每个账户的：
登录时间，用户IP，所访问的URL或IP，访问时间，传输的文件，流量、、、等等

多谢啦！

**Comment by puzzle9** [comment]
日志打印个 debug 就行
